File: frontend/src/Domain/Dtos/BehaviorGradeDto.ts
import EntityBase from '@/domain/EntityBase.ts';
import StudentDto from '@/domain/dtos/StudentDto.ts';
import TeacherDto from '@/domain/dtos/TeacherDto.ts';

interface BehaviorGradeDto extends EntityBase {
  value: number;
  student: StudentDto;
  teacher: TeacherDto;
}

export default BehaviorGradeDto;

File: Application/Dto/School/BehaviorGradeDto.cs
using Application.Dto.Users;
using Domain.Entities_v2.Types;

namespace Application.Dto.School;

public class BehaviorGradeDto: IBaseDto
{
    public int Id { get; set; }
    public DateTime Created { get; set; }
    public DateTime Updated { get; set; }
    public Status Status { get; set; }
    
    public int Value { get; set; }
    
    public StudentDto Student { get; set; } = null!;
    public TeacherDto Teacher { get; set; } = null!;
}